sklearn/inspection/partial_dependence.py

```diff
@@ -448,23 +458,37 @@ def plot_partial_dependence(estimator, X, features, feature_names=None,
         - 'auto':
           - 'recursion' is used for estimators that supports it.
           - 'brute' is used for all other estimators.
+
     n_jobs : int, optional (default=None)
         The number of CPUs to use to compute the partial dependences.
         ``None`` means 1 unless in a :obj:`joblib.parallel_backend` context.
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
+
     verbose : int, optional (default=0)
         Verbose output during PD computations.
+
     fig : Matplotlib figure object, optional (default=None)
         A figure object onto which the plots will be drawn, after the figure
         has been cleared. By default, a new one is created.
+
+        .. deprecated:: 0.22
+
     line_kw : dict, optional
         Dict with keywords passed to the ``matplotlib.pyplot.plot`` call.
         For one-way partial dependence plots.
+
     contour_kw : dict, optional
-        Dict with keywords passed to the ``matplotlib.pyplot.plot`` call.
+        Dict with keywords passed to the ``matplotlib.pyplot.contourf`` call.
         For two-way partial dependence plots.
 
+    ax : Matplotlib axes, list of Matplotlib axes or None, (default=None)
+        By default,
+
+    Returns
+    -------
+    sklearn.inspection.PartialDependenceDisplay
+
```

Comment:
Maybe update the example by showing it returns an object

Reply:
The object enables one to adjust the plot after the computation. This can not really be shown in the docstring, because we can not output the images.